runtime-js/numbers.js

```diff
@@ -216,6 +216,8 @@ atr$(JSNum$proto, 'character', function(){
   }
   return Character(this.valueOf());
 },undefined,function(){return{an:function(){return[shared(),actual()]},mod:$CCMM$,$cont:Integer,d:['$','Integer','$at','character']};});
+atr$(JSNum$proto, 'nearestFloat', function(){ return Float(this.valueOf()); },
+  undefined,function(){return{an:function(){return[shared(),actual()]},mod:$CCMM$,$cont:Integer,d:['$','Integer','$at','nearestFloat']};});
```

Comment:
Well I don't think that's the best way to do it. The "new" way to do it is to declare it in a separate file where Stitcher can find it.